applications/FluidDynamicsApplication/python_scripts/apply_slip_process.py

```diff
@@ -52,6 +48,11 @@ def __init__(self, Model, settings ):
                 1.0e8,
                 self.model_part.Nodes)
 
+    def ExecuteInitialize(self):
+        # Compute the normal on the nodes of interest -
+        # Note that the model part employed here is supposed to only have slip "conditions"
+        KratosMultiphysics.NormalCalculationUtils().CalculateOnSimplex(self.model_part, self.model_part.ProcessInfo[KratosMultiphysics.DOMAIN_SIZE])
```

Comment:
I'm wondering why did we place the normal calculation in the constructor instead of placing it here... I think it makes more sense to do it in the `Initialize`.